doc/source/reshaping.rst

```diff
@@ -698,10 +703,124 @@ handling of NaN:
     In [3]: np.unique(x, return_inverse=True)[::-1]
     Out[3]: (array([3, 3, 0, 4, 1, 2]), array([nan, 3.14, inf, 'A', 'B'], dtype=object))
 
-
 .. note::
     If you just want to handle one column as a categorical variable (like R's factor),
     you can use  ``df["cat_col"] = pd.Categorical(df["col"])`` or
     ``df["cat_col"] = df["col"].astype("category")``. For full docs on :class:`~pandas.Categorical`,
     see the :ref:`Categorical introduction <categorical>` and the
     :ref:`API documentation <api.categorical>`.
+
+Frequently Asked Questions (and Examples)
+------------------
+
+In this section, we will review frequently asked questions and examples. The 
+column names and relevant column values are named to correspond with how this
+DataFrame will be pivoted in the answers below.
+
+.. ipython:: python
+
+   np.random.seed([3,1415])
+   n = 20
+   
+   cols = np.array(['key', 'row', 'item', 'col'])
```

Comment:
you can just do
``
In [12]: cols + pd.DataFrame((np.random.randint(5, size=(n, 4)) // [2, 1, 2, 1]).astype(str))
Out[12]: 
       0     1      2     3
0   key1  row3  item2  col0
1   key0  row2  item1  col4
2   key1  row1  item0  col2
3   key1  row1  item0  col1
4   key0  row3  item1  col2
5   key1  row0  item2  col4
6   key2  row2  item0  col3
7   key2  row0  item2  col2
8   key1  row1  item0  col1
9   key0  row4  item0  col4
10  key0  row0  item1  col2
11  key0  row4  item1  col4
12  key0  row4  item2  col1
13  key1  row1  item1  col1
14  key1  row0  item2  col4
15  key2  row2  item1  col0
16  key2  row2  item2  col0
17  key0  row3  item0  col2
18  key1  row0  item1  col4
19  key0  row3  item1  col2
```

Reply:
Thanks! Refactored a bit.